build.gradle

```diff
@@ -143,6 +144,10 @@ dependencies {
     exclude(module: 'tools')
   }
 
+  compile("org.luaj:luaj-jse:${luajVersion}")
+  compile("com.github.micheljung:nocatch:${nocatchVersion}")
+  compile("commons-io:commons-io:${commonsIoVersion}")
```

Comment:
I've been avoiding commons-io since it overlaps with guava and JDK 7+ APIs a lot.